README.md

```diff
@@ -78,6 +78,14 @@ There are many more parameters you can use. Display the usage doc with the follo
 
     GitLink.exe -h
 
+### Native PDBs
+
+Native PDBs (from C++ projects) are supported by using -a option:
+
+    GitLink.exe <nativePdbfile> -a
+
+All source code files from your git depot will be indexed in the PDB.
```

Comment:
@AArnott That's why I proposed to keep it internal if we were able to detect the input pdb is native (which we don't for now).

Anyway we can make it obsolete as soon as we have a better C++ support.